apps/settings/elements/screen_lock.html

```diff
@@ -31,6 +31,15 @@ <h1 data-l10n-id="screenLock-header"></h1>
           </span>
         </li>
         <li class="passcode-enabled">
+          <p data-l10n-id="powerbutton-behavior">Powerbutton behavior</p>
```

Comment:
I would keep the string, conforming to the convention in this file. Will correct spelling and indent the HTML to avoid long lines.